app/services/outliers/calculate_bounding_boxes.rb

```diff
@@ -0,0 +1,64 @@
+class Outliers::CalculateBoundingBoxes
+  INITIAL_BOUNDING_BOX = {
+    min_latitude: "+90".to_d,
+    max_latitude: "-90".to_d,
+    min_longitude: "+180".to_d,
+    max_longitude: "-180".to_d,
+  }
+  INITIAL_BOUNDING_BOXES = {
```

Comment:
The name INITIAL_BOUNDING_BOXES doesn't seem the best place to contain number_of_outliers. But I don't have any idea how to call differently, so you can ignore me :)

Reply:
I feel you. Same here :)